cvmfs/cvmfs_config

```diff
@@ -99,6 +99,131 @@ var_list="$parm_list $switch_list"
 
 cvmfs_fs_bundle="/Library/Filesystems/cvmfs.fs/Contents/Resources" # Mac OS X
 
+PROCNAME=cvmfs2
+MOUNTPOINT=/cvmfs
+IS_KILLALL_STUCK=FALSE
+IS_KILLALL_FORCE=FALSE
```

Comment:
```suggestion
PROCNAME=cvmfs2
MOUNTPOINT=/cvmfs
IS_KILLALL_STUCK=FALSE
IS_KILLALL_FORCE=FALSE
```
I think it'd be better to avoid these global variables. PROCNAME and MOUNTPOINT are essentially const, and can just be hardcoded where they are used, that makes the code more readable.

To avoid  IS_KILLALL* you could call abort_*connections directly from parse_killall args.